egs2/wsj0-2mix/enh1/README.md

```diff
@@ -0,0 +1,57 @@
+<!-- Generated by ./scripts/utils/show_asr_result.sh -->
```

Comment:
Please replace it with actual enhancement/separation results.

Reply:
I have created `scripts/utils/show_enh_score.sh`, and it can show some basic enhancement/separation results.
`README.md ` has been updated.